src/img_web/img.conf

```diff
@@ -17,7 +17,7 @@ features_dir = %(doc_root)s%(url_prefix)s/features
 use_http_remote_user = no
 ; django secret key, generate one using
 ; django-admin generate_secret_key --settings=webhook_launcher.settings
-secret_key =
+secret_key = eFDqA6NjtbStLT0JHEowci1sClIzAdz6
```

Comment:
This was left empty on purpose so that things can't be run in production with the default key. On local testing you can have your own copy of the img.conf with the key filled in.